streaming/src/main/scala/org/apache/spark/streaming/receiver/ReceivedBlockHandler.scala

```diff
@@ -96,7 +96,7 @@ private[streaming] class BlockManagerBasedBlockHandler(
  */
 private[streaming] case class WriteAheadLogBasedStoreResult(
     blockId: StreamBlockId,
-    segment: WriteAheadLogFileSegment
+    segment: WriteAheadLogRecordHandle
```

Comment:
This is a leftover from the name change, segment --> handle. If there is no other change needed, I can take care of it in the next PR #5732